### PR TITLE
Cloning Youtube video page as HTML and CSS project at Microverse

### PR DESCRIPTION
**Pull request by anewman15**
We cloned Youtube video page as HTML and CSS project. The page contains a navbar with navigation icons, a search bar and a video player section with title, description and share and save options. It also has a recommended video section in the right side.

### CODE REVIEW

**Review by men32z — COMMENTED** [review]
Pull request currently under review by Luis Preza (TSE).

Kindly exercise patience 🐱

**Review by men32z — CHANGES_REQUESTED** [review]
Hey team!
## Good job on this project! 👍

There are still some changes 👇 you need to implement to approve this request 🔍

* [x]  [REQUIRED] The search bar is aligned to the middle of the navbar ([screenshot ](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/embedding_images_and_video/searchbar_align_requirement.png)from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/48945837/))
* [x] [OPTIONAL] The project has a descriptive README.md ([screenshot ](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/descriptive_readme.png)from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))
you have a descriptive read me but you could improve it more, for example adding the live version you could use [https://raw.githack.com/](https://raw.githack.com/)
* [ ] [REQUIRED]  Pull Request has meaningful title and a short description ([screenshot ](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/descriptive_prs.png)from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))

- [ ] [REQUIRED] Check HTML markup with this validator https://validator.w3.org/#validate_by_upload. (Students were asked to use this one, so in most cases their code should be already validated) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/validation_requirement.png) from Pathwright)

- [x] [OPTIONAL] Check usage of [semantic HTML](https://www.w3schools.com/html/html5_semantic_elements.asp) elements. They should be "appropriately sized and positioned" (as described in Odin project description) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/semantic_html_requirement.png) from Pathwright)

you are usign currently the HTML5 sematinc but you could improve this by adding main and footer tags propertly. 
[`<main>` tag](https://www.w3schools.com/tags/tag_main.asp) for the dominant content of the `<body>` of a document and check
 [`<footer>`](https://www.w3schools.com/TagS/tag_footer.asp)  since is not required in this project just take a look.

## Please, make all REQUIRED changes and submit your request again! 😀

If you have any questions, feel free to send me a message through Slack (@men32z)
I would be happy to help! 👩‍💻

Happy coding 🤓

--Luis Preza

File: styles/main-css.css [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+  display: flex;
+  justify-content: start;
+  
+}
+.material-icons{
+
+}
+.header-search-bar{
+  width: 80%;
+}
```
Comment (men32z) [inline styles/main-css.css]: * [x] [REQUIRED] The search bar is aligned to the middle of the navbar  you could achieve this by using the following properties in the "header-search-bar" class
```
width: 100%;
text-align: center; 
```

**Comment by anewman15** [comment]
Hi Luis,

Thanks for providing your feedback. We changed our code and pull request as you suggested. Please let us know what you think.

My best,
Abdullah and Asghar

________________________________

**Comment by adamclasic** [comment]
@anewman15 @beyk 
You should request a code rev again if changes are required.

**Review by men32z — CHANGES_REQUESTED** [review]
Hey team!
## Good job making the changes on this project! 👍 you almost got it
There are still some changes 👇 you need to implement to approve this request 🔍

#### Pull Request description should be something that describes the features that it brings. for instance "Web page clone of youtube, it has a navbar with a search input, i shows a video. etc..." you can modify the PR description without make a new one.
- [ ] [REQUIRED]  Pull Request has meaningful title and a short description ([screenshot ](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/descriptive_prs.png)from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))

#### you have to upload your html file to check HTML errrors, let me know if you have problems with this.
- [ ] [REQUIRED] Check HTML markup with this validator https://validator.w3.org/#validate_by_upload. (Students were asked to use this one, so in most cases their code should be already validated) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/validation_requirement.png) from Pathwright)



- [ ] [OPTIONAL] the name of the main html file should be by standart "index.html" is a convention.
- [ ] [OPTIONAL] fix the stickler errors, it is not required in this project but could be a good practice. 

## Please, make all REQUIRED changes and submit your request again! 😀
If you have any questions, feel free to send me a message through Slack (@men32z)
I would be happy to help! 👩‍💻
Happy coding 🤓
--Luis Preza

**Comment by anewman15** [comment]
Thanks! We have made the suggested changes.

**Review by serdg0 — APPROVED** [review]
**Status: Approved.**

### Great Job! :100: :muscle: 
Go on and merge :+1: 
Keep the great work in the next project :muscle: 

### Happy Coding! :computer: :computer:

**Comment by anewman15** [comment]
Thanks, @serdg0 and @men32z !

**Pull request merged**